Firebase/android/app/src/main/java/com/novoda/bonfire/chat/view/ChatAdapter.java

```diff
@@ -30,11 +33,17 @@ public void update(Chat chat, User user) {
 
     @Override
     public MessageViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        MessageBubbleDrawable bubbleDrawable;
+        MessageView messageView;
         if (viewType == 0) {
```

Comment:
Could I recommend extracting a `VIEW_TYPE_MESSAGE_OTHER_USERS` maybe? :) Also a `VIEW_TYPE_MESSAGE_THIS_USER` and use it here (`else if` it), so you have an `else` that can throw